### PR TITLE
Oauth google

### PR DESCRIPTION
**Pull request by f00se**
Task #2

Nothing what I remember now to mention.

### CODE REVIEW

File: spec/models/user_spec.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+describe User, type: :model do
```
Comment (PoslinskiNet) [inline spec/models/user_spec.rb]: Empty spec?

File: spec/support/omniauth_helper.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# module OmniauthHelper
```
Comment (PoslinskiNet) [inline spec/support/omniauth_helper.rb]: Redundant?

File: test/controllers/front_controller_test.rb [hunk 3]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class FrontControllerTest < ActionDispatch::IntegrationTest
```
Comment (PoslinskiNet) [inline test/controllers/front_controller_test.rb]: Redundant?

File: test/fixtures/users.yml [hunk 4]
```diff
@@ -0,0 +1,11 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
```
Comment (PoslinskiNet) [inline test/fixtures/users.yml]: Redundant?

File: test/models/user_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (PoslinskiNet) [inline test/models/user_test.rb]: Redundant?

Comment (f00se) [inline test/models/user_test.rb]: Can I disable auto generating tests in 'test' directory?

**Comment by f00se** [comment]
Done

File: spec/rails_helper.rb [hunk 6]
```diff
@@ -5,6 +5,11 @@
 abort('The Rails environment is running in production mode!') if Rails.env.production?
 require 'spec_helper'
 require 'rspec/rails'
+require 'simplecov'
+
+Dir[Rails.root.join('spec', 'support', '**', '*.rb')].each { |f| require f }
+
+SimpleCov.start
```
Comment (PoslinskiNet) [inline spec/rails_helper.rb]: Generate specs only if `ENV['COVERAGE']` is set.

**Comment by f00se** [comment]
Travis has error: Could not load database configuration. No such file
This is because database.yml is only in local distribution. I don't know what to do with this. Maybe I should override something.

**Pull request merged**